tools/update-quarkus-versions.sh

```diff
@@ -1,21 +1,21 @@
 #!/bin/sh
```

Comment:
you have mad `/bin/[ba]sh` skillz. Arrays may be a bash-ism?

probably better to 

```suggestion
#! env bash
```